src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -47,9 +47,17 @@ async function boot(userOptions?: Partial<BlazorOptions>): Promise<void> {
     return true;
   };
 
+  window.addEventListener('unload', sendDisconnect, false);
+
   window['Blazor'].reconnect = reconnect;
 
   logger.log(LogLevel.Information, 'Blazor server-side application started.');
+
+  function sendDisconnect(_: Event): void {
+    const data = new FormData();
+    data.set('circuitId', circuit.circuitId);
+    navigator.sendBeacon(`${document.baseURI}_blazor/disconnect`, data);
```

Comment:
Is it technically necessary to supply an absolute URL for `sendBeacon`? Does this fail if it was just:

```ts
navigator.sendBeacon(`_blazor/disconnect`, data);
```

I can't see anything in the spec that seems to imply this wouldn't work. The example [here](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon) uses a relative URL.

Reply:
Fair enough, I just put this together before doing cleanup.